.gitlab/bazel/build-deps.yaml

```
@@ -7,9 +7,17 @@
 
 bazel:build-deps:linux-amd64:
   extends: [ .bazel:build-deps, .linux_x64 ]
-  tags: [ "arch:amd64", "specific:true" ]
+  tags: ["linux-persistent:amd64", "specific:true"]
+  cache: []
   script:
+  # Wrapper in tools/bazel expects these directories to exist
+  # TODO(agent-build): make the wrapper to create these directories
+    - mkdir -p /data/bzl /data/output /data/bzl-repo
     - bazel build //deps/...
+  variables:
+      BAZEL_DISK_CACHE: "/data/bzl"
+      BAZEL_OUTPUT_USER_ROOT: "/data/output"
+      BAZEL_REPO_CONTENTS_CACHE: "/data/bzl-repo"
```

I guess we would simplify our life with just:
```suggestion
  variables:
      XDG_CACHE_HOME: /data/bzl
```

---

Also: `--repo_contents_cache` != `--repository_cache` (if I guess well the intended purpose of `bzl-repo`)

---

that's kind of convoluted:
- baselisk's cache would be under bazel's output user root
- bazel's **repo contents** cache would be under what's look like intended for bazel's **repository** cache
- bazel's repository cache will effectively land in a subdirectory of bazel's output user root

what was wrong with only `XDG_CACHE_HOME: /data/bzl`?